pydax/_high_level.py

```diff
@@ -33,6 +33,17 @@ def get_config() -> Config:
     """Returns global PyDAX configs.
 
     :return: Read-only global configs represented as a data class
+
+    Example get_config:
```

Comment:
``Example:`` should be sufficient, this is in the docstring of `get_config`, so it would only show under the function of `get_config`.